packages/master-detail-layout/theme/material/vaadin-master-detail-layout-styles.js

```diff
@@ -0,0 +1,29 @@
+import '@vaadin/vaadin-material-styles/color.js';
+import '@vaadin/vaadin-material-styles/style.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+registerStyles(
+  'vaadin-master-detail-layout',
+  css`
+    :host(:is([overlay], [stack])) [part='detail'] {
+      background-color: var(--material-background-color);
+    }
+
+    :host([overlay]) [part='detail'] {
+      box-shadow: var(--material-shadow-elevation-4dp);
+    }
+
+    :host([overlay][orientation='horizontal']) [part='detail'] {
+      border-inline-start: 1px solid var(--material-divider-color);
+    }
+
+    :host([overlay][orientation='vertical']) [part='detail'] {
+      border-block-start: 1px solid var(--material-divider-color);
+    }
+
+    :host([overlay]) [part='backdrop'] {
+      background-color: var(--material-secondary-background-color);
```

Comment:
This seems to cover the master area completely. Do we have a semi-transparent color? Or should we add opacity?

Reply:
There are no semi-transparent background colors in Material theme. Added `opacity: 0.5`, maybe that could work.